CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # True Changelog
 
+## UNRELEASED
```

Comment:
Should probably note the new API arguments more clearly here?

Reply:
Better? https://github.com/oddbird/true/pull/221/commits/3a89271b59e273fa8e4670f7c63afbce108560bb

Reply:
lgtm